definitions/benefit/sheets/CaseEventToComplexTypes/CaseEventToComplexTypes.json

```diff
@@ -5048,6 +5209,27 @@
     "FieldDisplayOrder": 7,
     "PageID": "1.0"
   },
+  {
+    "CaseEventID": "incompleteApplicationReceived",
```

Comment:
we were trying to get into a practice where if we're updating an event we'll move it to its own file, so that gradually we'll make working with the JSON more manageable than this ginormous files (not a must, just if you can)